config.yaml

```diff
@@ -14,6 +14,16 @@ options:
     default: "192.168.0.1"
     description: |
       Default gateway for the pod network
+  node-switch-cidr:
+    type: string
+    default: "100.64.0.0/16"
+    description: |
+      CIDR for node switch (This value cannot be changed after deployment).
```

Comment:
Is there any code to prevent the user from changing them?

Reply:
oh well in that case, i'm alright.  I didn't know if the user might change this and wonder why nothing happened.  I'm guessing this description is good enough for now